### PR TITLE
Experimental UI

### LINKED ISSUE #28: Experimental UI
@PetalCat / @ParkerH27 is currently building a new UI that doesn't use `w3.css`.
```[tasklist]
## TODO List
- [x] Wipe Styles
- [x] Begin using Tailwind via the [CDN](https://tailwindcss.com/docs/installation/play-cdn)
- [ ] Remove support help page.... Directions are enough
- [ ] Remove Easter Egg (partially done)
- [x] Add new #20
- [x] Add new backgrounds
- [ ] Add background switching Easter Egg
- [ ] Update theming to match backgrounds
- [ ] Use the Tailwind [CLI](https://tailwindcss.com/docs/installation) with [Vite](https://tailwindcss.com/docs/guides/vite)
```

### PR DESCRIPTION
**Pull request by lishaduck**
Use [Tailwind](https://tailwindcss.com/), for @ParkerH27.

### CODE REVIEW

File: src/styles/fonts/norwester.otf
Comment (lishaduck) [inline src/styles/fonts/norwester.otf:1]: See #20.

**Comment by lishaduck** [comment]
Does [soanvig/mm-js](https://github.com/soanvig/mm-js)r help, @ParkerH27?

**Comment by lishaduck** [comment]
Does switching to TS still break stuff?

**Comment by lishaduck** [comment]
At some point, do we want to turn on [XO](https://github.com/xojs/xo)?

**Comment by lishaduck** [comment]
> At some point, do we want to turn on [XO](https://github.com/xojs/xo)?

No, we don't. Base ESLint is much better.

**Review by ParkerH27 — APPROVED** [review]

**Comment by lishaduck** [comment]
We should use Netlify if I can get it to work, @ParkerH27, because of deploy previews. However, doing so means that we can't use GH Pages anymore. Is that OK with you?
Current Netlify URLs:
- https://pattonville-wallet.netlify.app/
- https://pattonville-gpa.netlify.app/
- https://phs-map.netlify.app/

**Comment by PetalCat** [comment]
Is it possible to still keep the GitHub pages and have that URL redirect to the new hosting?

**Comment by PetalCat** [comment]
Or what I mean by that is if people used the old URL it would just go to the new one

**Comment by lishaduck** [comment]
Should be. Check the docs.

**Comment by lishaduck** [comment]
Tada, @ParkerH27 - [StackOverflow](https://stackoverflow.com/a/66602291) has a solution. And I fixed the build. 😁

**Comment by lishaduck** [comment]
Can you fix `tile.png`'s png signature, @ParkerH27? I can't preview it in VCode because Luna Paint requires valid signatures.

**Comment by ParkerH27** [comment]
> Can you fix `tile.png`'s png signature, @ParkerH27? I can't preview it in VSCode because `Luna Paint` requires valid signatures.

You can just delete tile.png... it is unused

**Comment by lishaduck** [comment]
The PWA starter's been updated. Should it get merged into this, @ParkerH27?

**Comment by lishaduck** [comment]
So, it was e34f0c8476401c0e24971815938ec7973b70500d that did it.

**Comment by lishaduck** [comment]
@ParkerH27, @PetalCat, IT WORKS!

File: src/styles/fonts/norwester.otf
Comment (lishaduck) [reply src/styles/fonts/norwester.otf:1]: Norwester isn't on GFonts, so nevermind.

**Pull request merged**